client/src/components/ActivitiesList.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+
+const ActivitiesList = (props) => {
+  const [activities, setActivities] = useState([]);
+  const [category, setCategory] = useState({});
+
+  const getActivities = async () => {
+    try {
+      const response = await fetch(`/api/v1/categories/${props.computedMatch.params.id}`);
```

Comment:
what is this `computedMatch` property? would generally expect this to be `props.match.params.id`

Reply:
That was just how the props came out of the Authenticated Router